### PR TITLE
Added check that nu must be a scalar in MvStudentTRV

### PR DESCRIPTION
**Pull request by sagartomar**
This should fix issue #5214

### CODE REVIEW

File: pymc/distributions/multivariate.py [hunk 1]
```diff
@@ -265,6 +265,13 @@ class MvStudentTRV(RandomVariable):
     dtype = "floatX"
     _print_name = ("MvStudentT", "\\operatorname{MvStudentT}")
 
+    def make_node(self, rng, size, dtype, nu, mu, cov):
+        nu = at.as_tensor_variable(floatX(nu))
```
Comment (ricardoV94) [inline pymc/distributions/multivariate.py]:
```suggestion
        nu = at.as_tensor_variable(nu)
```
I think we don't need the `floatX` here, that is something that belongs to PyMC distributions.

**Review by ricardoV94 — CHANGES_REQUESTED** [review]
Looks great. I left a suggestion below, and we should add a test that covers this branch.

Perhaps in `test_distributions_random` you can add something along the lines of the following one:

https://github.com/pymc-devs/pymc/blob/95bd5e59f940cfbd4076dd21d93b0e2aec270add/pymc/tests/test_distributions_random.py#L1622-L1632

**Review by michaelosthege — APPROVED** [review]
Looks good!
I like the `re.escape(msg)` it always takes me ages to find the right escape syntax..

**Review by ricardoV94 — APPROVED** [review]
Looks great

**Pull request merged**